galactic/distribution.yaml

```diff
@@ -4329,6 +4329,21 @@ repositories:
       url: https://github.com/ros-visualization/rqt_topic.git
       version: foxy-devel
     status: maintained
+  rslidar_msg:
+    doc:
+      type: git
+      url: https://github.com/RoboSense-LiDAR/rslidar_msg.git
+      version: master
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/nobleo/rslidar_msg-release.git
+      version: 0.0.0-1
```

Comment:
Is this version correct? Based on the source code I would have expected 1.3.0: https://github.com/RoboSense-LiDAR/rslidar_msg/blob/1a416d44c0132230adc781e8768954865f3b8c49/package.xml#L5

Let me know if you'd like to proceed with this PR or re-bloom the package.

Reply:
Ah yes, I need to re-bloom now I fixed the version number!